autogluon/utils/tabular/ml/models/knn/knn_model.py

```diff
@@ -32,7 +32,7 @@ def preprocess(self, X):
     def _set_default_params(self):
         default_params = {
             'weights': 'uniform',
-            'n_jobs': -1,
+            'index_factory_string' : 'Flat',
```

Comment:
Are we not able to use multiple cores with this implementation? Why was n_jobs removed?

Reply:
Added this param back in to allow for the use of fewer cores. It was removed because Faiss uses OpenMP underneath, which defaults to using all cores and there wasn't much in the docs on how to change it